dashboard/app/assets/javascripts/dashboard.controller.js

```diff
@@ -0,0 +1,49 @@
+(function() {
+  'use strict';
+
+  angular.module('qat-dashboard').controller('DashboardController', [
+    'testExecutor',
+    function(testExecutor) {
+      var vm = this;
```

Comment:
@mDibyo Instead of referencing `this`, I would recommend doing the following at the bottom:

```
return {
  testExecsMap: testExecutor.all,
  // ...
}
```